### DISCUSSION TITLE
How to hide row details in a `DataGrid` on click (on the same row)

### DISCUSSION THREAD

**Discussion by dend** [microsoft/microsoft-ui-xaml#9314] [Q&A]
I have a `DataGrid` control in my app that shows details whenever the user clicks on it, with the help of `RowDetailsVisibilityMode="VisibleWhenSelected"`. It all works splendidly, however I want to hide the details when the user clicks on the exact same row (that is - selection doesn't change, as `SelectionChanged` event is not fired in that case). What's the best way to hide them? Documentation doesn't seem to cover this scenario.

**Comment by kmgallahan** [comment]
WinUI as found in this repo doesn't have a DataGrid.
